test/controllers/caseload_controller_test.exs

```diff
@@ -0,0 +1,25 @@
+defmodule Healthlocker.CaseloadControllerTest do
+  use Healthlocker.ConnCase
+
+  alias Healthlocker.{EPJSTeamMember, EPJSUser, ReadOnlyRepo, User}
+
+  describe "current_user is assigned in the session" do
+    setup do
+      %User{
+        id: 123456,
+        name: "MyName",
+        email: "abc@gmail.com",
+        password_hash: Comeonin.Bcrypt.hashpwsalt("password"),
+        security_question: "Question?",
+        security_answer: "Answer"
+      } |> Repo.insert
+
+      {:ok, conn: build_conn() |> assign(:current_user, Repo.get(User, 123456)) }
+    end
+
+    test "/caseload :: index", %{conn: conn} do
```

Comment:
The usual style for writing naming these controller tests names would be `"GET /caseload"`.